doc/cephfs/cephfs-mirroring.rst

```diff
@@ -44,6 +44,23 @@ The mirror daemon should be spawned using `systemctl(1)` unit files::
 
 .. note:: The user specified here is `mirror` created in the `Creating Users` section.
 
+Multiple `cephfs-mirror` daemons can be spawned for concurrent synchronization and high
+availability. Mirror daemons would share the synchronization load amongst themselves
+using a simple `M/N` policy, where `M` is the number of directories and `N` is the number
+of `cephfs-mirror` daemons.
+
+When cephadm is used to manage a Ceph cluster, `cephfs-mirror` daemons can be deployed using::
+
```

Comment:
We aren't super consistent about this, but I suggest `cephadm` in backquotes.

Reply:
Won't double backquotes be better than single backquotes in this case? Single backquotes just render the text as italics whereas double backquotes would reder it with monospace font thus emphasizing it as a literal.

Strange that single backquotes is same as italics. It ought to be used for something more/better.

Reply:
I've always used single backquotes in the docs I've written.